pkg/authorization/apis/authorization/rbacconversion/conversion.go

```diff
@@ -315,3 +316,36 @@ func determineGroupKind(group string) string {
 	}
 	return kind
 }
+
+func RBACSubjectsToUsersAndGroups(subjects []rbac.Subject, defaultNamespace string) (users []string, groups []string) {
+	for _, subject := range subjects {
+		if subject.APIGroup != rbac.GroupName {
```

Comment:
misses service account subjects, right?

Reply:
> misses service account subjects, right? is there a test that would have caught that?

betting we'll find out.

Reply:
This goes in `pkg/authorization/util/subject.go`

Reply:
> misses service account subjects, right? is there a test that would have caught that?
> betting we'll find out.

Yes, there are integration tests that fail when this is wrong.